File: temp_eval.js
const fs = require('fs');
const env = fs.readFileSync('.env', 'utf-8');
const key = env.split('\n').find(line => line.startsWith('GROQ_API_KEY=')).split('=')[1].trim();

fetch('https://api.groq.com/openai/v1/models', {
  headers: {
    'Authorization': `Bearer ${key}`
  }
}).then(res => res.json()).then(console.log).catch(console.error);

File: temp_eval.py
import os, requests
from dotenv import load_dotenv
load_dotenv('.env')
key = os.getenv('GROQ_API_KEY')
print(f'Key starts with: {key[:10]}...')
res = requests.get('https://api.groq.com/openai/v1/models', headers={'Authorization': f'Bearer {key}'})
print(res.status_code)
print(res.json())
